ethcore/src/spec/spec.rs

```diff
@@ -65,6 +67,7 @@ impl From<ethjson::spec::Params> for CommonParams {
 			subprotocol_name: p.subprotocol_name.unwrap_or_else(|| "eth".to_owned()),
 			min_gas_limit: p.min_gas_limit.into(),
 			fork_block: if let (Some(n), Some(h)) = (p.fork_block, p.fork_hash) { Some((n.into(), h.into())) } else { None },
+			eip98_transition: p.eip98_transition.map_or(0, Into::into),
```

Comment:
Wouldn't that enable `eip98` automatically for spec files where that prop is not defined (private chains?)?